lib/services/profiles-api-service.js

```diff
@@ -141,12 +140,16 @@ function profileApiServiceFactory(
         if (node.type === 'switch') {
             configuration = self.getSwitchDiscoveryConfiguration(node, options.switchVendor);
         } else {
+            var setObm = configFile.get('obmOption', 'false');
             configuration = {
                 name: 'Graph.SKU.Discovery',
                 options: {
                     defaults: {
                         graphOptions: {
-                            target: node.id
+                            target: node.id,
+                            'obm-option' : {
```

Comment:
The same suggestion in https://github.com/RackHD/on-http/pull/544, change the variable to be a better name.